Remove the bundled HTSlib build and use an EasyBuild dependency.
Author: Emik Lin (HKUMed CPOS)
diff --git a/Makefile b/Makefile
index cd2dcab..68c686e 100644
--- a/Makefile
+++ b/Makefile
@@ -3,8 +3,8 @@ ARCH := $(shell arch)

 PYTHON ?= python3

-all : libhts.a longphase libclair3.so
-clean : clean_htslib clean_longphase clean_libclair3
+all : longphase libclair3.so
+clean : clean_longphase clean_libclair3

 SAMVER	=	1.15.1
 PREFIX	?=	${CONDA_PREFIX}
@@ -32,18 +32,0 @@
-samtools-$(SAMVER)/Makefile:
-	curl -L -o samtools-${SAMVER}.tar.bz2 https://github.com/samtools/samtools/releases/download/${SAMVER}/samtools-${SAMVER}.tar.bz2
-	tar -xjf samtools-${SAMVER}.tar.bz2
-	rm samtools-${SAMVER}.tar.bz2
-
-libhts.a: samtools-$(SAMVER)/Makefile
-	# this is required only to add in -fpic so we can build python module
-	@echo "\x1b[1;33mMaking $(@F)\x1b[0m"
-	cd samtools-${SAMVER}/htslib-${SAMVER}; \
-	if [ "$(OS)" = "Darwin" ]; then \
-		cp -f ${BUILD_PREFIX}/share/gnuconfig/config.* . && autoheader && autoconf -Wno-syntax && CC=${CC_PATH} CPPFLAGS="" CFLAGS="-O3 -fpic" LDFLAGS="" ./configure; \
-	else \
-		CFLAGS="${CFLAGS}" LDFLAGS="${LDFLAGS}" ./configure; \
-	fi; \
-	make CFLAGS="${CFLAGS}" LDFLAGS="${LDFLAGS}"
-	cp samtools-${SAMVER}/htslib-${SAMVER}/$@ $@
-
-
@@ -64 +46 @@ longphase:
-libclair3.so: samtools-${SAMVER}/htslib-${SAMVER} libhts.a
+libclair3.so:
@@ -77,6 +59,0 @@ fa_test: src/fa_test.o src/levenshtein.o src/medaka_bamiter.o src/medaka_common.
-.PHONY: clean_htslib
-clean_htslib:
-	cd samtools-${SAMVER} && make clean || exit 0
-	cd samtools-${SAMVER}/htslib-${SAMVER} && make clean || exit 0
-	rm libhts.a
-
